src/Test/Integration/Model/Service/ImportOrderTest.php

```diff
@@ -72,13 +82,83 @@ public function testOrderImport(\ShopgateOrder $order)
         $this->orderHolder[] = $result;
     }
 
+    /**
+     * @param string $expectedPaymentCode
+     * @param array  $paymentInformation
+     * @param string $paymentMethod
+     * @param string $paymentGroup
+     *
+     * @throws ShopgateLibraryException
+     *
+     * @dataProvider         paymentDataProvider
+     * @magentoConfigFixture default/payment/shopgate/order_status processing
+     * @magentoConfigFixture current_store payment/braintree/active 1
+     */
+    public function testPaymentMapping(
+        string $expectedPaymentCode,
+        array $paymentInformation,
+        string $paymentMethod,
+        string $paymentGroup
+    ): void {
+        $shopgateOrder = new \ShopgateOrder(
+            [
+                'order_number'               => rand(1000000000, 9999999999),
```

Comment:
don't forget to add the **throws**

```suggestion
                'order_number'               => random_int(1000000000, 9999999999),
```